gpm_api/tests/test_visualization/test_facetgrid.py

```diff
@@ -0,0 +1,229 @@
+        orbit_dataarray_x6: xr.DataArray,
+    ) -> None:
+        """Test plotting orbit data using col argument"""
+
+        p = plot.plot_map(orbit_dataarray_x6, col=CHANNEL, col_wrap=3)
+        p.set_title("Title")
+        save_and_check_figure(figure=p.fig, name=get_test_name())
+
+    def test_orbit_row(
+        self,
+        orbit_dataarray_x6: xr.DataArray,
+    ) -> None:
+        """Test plotting orbit data using row argument"""
+
+        p = plot.plot_map(orbit_dataarray_x6, row=CHANNEL, col_wrap=3)
+        save_and_check_figure(figure=p.fig, name=get_test_name())
+
+    def test_orbit_col_row(
+        self,
+        orbit_dataarray: xr.DataArray,
+    ) -> None:
+        """Test plotting orbit data using row and col arguments"""
+
+        channel1 = "channel1"
+        channel2 = "channel2"
+
+        orbit_dataarray = expand_dims(orbit_dataarray, 2, channel=channel1)
+        orbit_dataarray = expand_dims(orbit_dataarray, 2, channel=channel2)
+        # Calling with call_wrap=1, which should be ignored
+        p = plot.plot_map(orbit_dataarray, col=channel1, row=channel2, col_wrap=1)
+        save_and_check_figure(figure=p.fig, name=get_test_name())
+
+    def test_orbit_no_col_row(
+        self,
+        orbit_dataarray_x4: xr.DataArray,
+    ) -> None:
+        """Test plotting orbit data without valid col or row argument"""
+
+        with pytest.raises(ValueError):
+            plot.plot_map(orbit_dataarray_x4, col=None)
+
+    def test_orbit_one_empty_subplot(
+        self,
+        orbit_dataarray: xr.DataArray,
+    ) -> None:
+        """Test plotting orbit data with one empty subplot"""
+
+        orbit_dataarray = expand_dims(orbit_dataarray, 3, channel=CHANNEL)
+        p = plot.plot_map(orbit_dataarray, col=CHANNEL, col_wrap=2)
+        save_and_check_figure(figure=p.fig, name=get_test_name())
+
+    def test_orbit_extent(
+        self,
+        orbit_dataarray_x4: xr.DataArray,
+    ) -> None:
+        """Test plotting orbit data while specifying extent"""
+
+        extent = [0, 20, 0, 20]
+        p = plot.plot_map(orbit_dataarray_x4, col=CHANNEL, col_wrap=2)
+        p.remove_title_dimension_prefix()
+        p.set_extent(extent)
+        save_and_check_figure(figure=p.fig, name=get_test_name())
+
+    def test_orbit_non_unique_index(
+        self,
+        orbit_dataarray_x4: xr.DataArray,
+    ) -> None:
+        """Test plotting orbit data with non-unique channel index"""
+
+        orbit_dataarray_x4[CHANNEL] = [0, 1, 1, 2]
+        with pytest.raises(ValueError):
+            plot.plot_map(orbit_dataarray_x4, col=CHANNEL, col_wrap=2)
+
+    def test_orbit_no_colorbar(
+        self,
+        orbit_dataarray_x4: xr.DataArray,
+    ) -> None:
+        """Test plotting orbit data without colorbar"""
+
+        p = plot.plot_map(orbit_dataarray_x4, col=CHANNEL, col_wrap=2, add_colorbar=False)
+        save_and_check_figure(figure=p.fig, name=get_test_name())
+
+    def test_orbit_horizontal_colorbar(
```

Comment:
I think there is not a test unit for a plot_map or plot_image (without facetgrid) with a horizontal colorbar !